scrapers/Clips4Sale.yml

```diff
@@ -6,40 +6,56 @@ sceneByURL:
     scraper: sceneScraper
 xPathScrapers:
   sceneScraper:
-    common:
-      $studio: //span[contains(text(),"From:")]/following-sibling::a
     scene:
-      Title: //div[@data-clipid]/following-sibling::h3
+      Title: 
+        selector: //div[@class="clip-info__title"]/h3/text()
+        postProcess:
+          - replace:
+              - regex: \s*MP4
+                with:
+              - regex: \s*Optimum
+                with:
+              - regex: \s*OPTIMUM
+                with:
+              - regex: \s*WMV
+                with:
+              - regex: \s*\(HD-1080p\)
+                with:
+              - regex: \s*\(HD-720p\)
+                with:
```

Comment:
Good point, we can expand that further.
Example: https://regex101.com/r/CYLeuO/1